resources/css/style.css

```diff
@@ -45,9 +45,22 @@
 .title-get-involved {
   font-family: -apple-system, BlinkMacSystemFont, Roboto, sans-serif;
   font-size: 5em;
-  font-weight: 100;
+  font-weight: 300;
   text-align: center;
 }
+.row .description {
```

Comment:
This is changing the alignment of description in HOME page too, so we might need  a new class here or different set of selectors